### DISCUSSION TITLE
Is there anyway to show which line a tokio task is running at in tokio console or other tools?

### DISCUSSION THREAD

**Discussion by MrCroxx** [tokio-rs/tokio#4704] [Q&A] [answered]
Sorry to bother, it's me again. I'm diagnosing a deadlock / bad channel issue in my project which used tokio as runtime. I tried tokio-console to show resourses of `tokio::sync` but found everything looks normal. I think it's useful to show which line that a tokio task is running at. Is this feature supported by tokio-console or other tools?

Thanks a lot. ❤️

**Comment by Darksonn** [comment, marked as answer]
No, I don't think that information is supported by any tool. Unfortunately, I don't think Rust even makes it possible for tokio-console (or other tools) to query that information in the first place.

**Reply by MrCroxx** [reply]
Thanks for your patient answer. I'm not familiar with PL theory. But it really helps if supported. ❤
